[msx] testStarted testDFSAdminUnreachableDatanode
2019-12-19 18:54:17,338 [main] INFO  namenode.NameNode (NameNode.java:<init>(946)) - [msx-restart] reconfigMode is none, do nothing
2019-12-19 18:54:17,479 [main] INFO  namenode.NameNode (NameNode.java:<init>(1016)) - [msx-restart] NameNode start HostAndPort localhost:39291
2019-12-19 18:54:17,488 [main] INFO  datanode.DataNode (DataNode.java:<init>(464)) - [msx-restart] DataNode new conf
2019-12-19 18:54:17,550 [main] INFO  datanode.DataNode (DataNode.java:runDatanodeDaemon(2653)) - [msx-restart] DataNode start
2019-12-19 18:54:17,573 [main] INFO  datanode.DataNode (DataNode.java:<init>(464)) - [msx-restart] DataNode new conf
2019-12-19 18:54:17,646 [main] INFO  datanode.DataNode (DataNode.java:runDatanodeDaemon(2653)) - [msx-restart] DataNode start
2019-12-19 18:54:17,814 [main] INFO  datanode.DataNode (DataNode.java:shutdown(1995)) - [msx-restart] DataNode stop
2019-12-19 18:54:17,869 [main] INFO  datanode.DataNode (DataNode.java:shutdown(1995)) - [msx-restart] DataNode stop
2019-12-19 18:54:17,930 [main] INFO  namenode.NameNode (NameNode.java:stop(1059)) - [msx-restart] NameNode stop HostAndPort localhost:39291 double check before stop dfs.namenode.audit.log.async = false
[msx] testFinished testDFSAdminUnreachableDatanode testresult=1
